.github/scripts/performance-tests/produce_metric_widget_images.py

```diff
@@ -366,17 +366,22 @@ def parse_args():
             f"Will create a snapshot at URL: https://github.com/{args.github_repository}/blob/gh-pages/{snapshot_location}",
         )
 
-    # Delete oldest snapshots
-
-    snapshot_dirs_length = len(os.listdir(SOAK_TESTS_SNAPSHOTS_COMMITS_DIR))
-
-    if snapshot_dirs_length > args.max_benchmarks_to_keep:
-        oldest_snapshot_dirs = nsmallest(
-            snapshot_dirs_length - args.max_benchmarks_to_keep,
-            Path(SOAK_TESTS_SNAPSHOTS_COMMITS_DIR).iterdir(),
-            key=os.path.getmtime,
-        )
-        for old_snapshots_dir in oldest_snapshot_dirs:
-            shutil.rmtree(old_snapshots_dir, ignore_errors=True)
+    # Delete oldest run folders in most recent commit and oldest commit folders
+
+    for snapshots_dir in [
```

Comment:
Can you split the change to auth and the other edits into separate PRs?

Reply:
Sure, can do!

Reply:
Moved authentication updates to #88